### ISSUE TITLE
Launch at Login Not working

### ISSUE THREAD

**Issue by unexpectedreboot88** [nielsmouthaan/ejectify-macos#59]
I installed Ejectify 1.2.1 on my M1 Mac mini today. It's a fresh copy of macOS Ventura 13.4. Despite clicking the "launch at login" option in the menu, Ejectify will not launch on login no matter what I try. Oddly, when I installed it on my MacBook running 13.4, I can't recreate the issue. 

On my Mac mini, Ejectify shows up in the "allow in background section" of System Settings. If there's anything I can do to help (i.e. gather logs, etc.), please let me know.

Thank you,
Jacob

**Comment by nielsmouthaan** [comment]
Is the app copied to your Applications folder?

**Comment by unexpectedreboot88** [comment]
Yes, it is in the applications folder. On May 22, 2023, at 6:22 AM, Niels Mouthaan ***@***.***> wrote:﻿
Is the app copied to your Applications folder?

**Comment by nielsmouthaan** [comment]
Very weird. Can you [manually mark the app to be opened at launch](https://support.apple.com/guide/mac-help/open-items-automatically-when-you-log-in-mh15189/mac)?

**Comment by unexpectedreboot88** [comment]
I tried to add Ejectify to the Open at Login section by clicking the + button, then browsing to the Applications folder and selecting the app. Once I select Ejectify and click Open, the add application search window disappears and takes me back to the Open at Login list that is still empty. 

Oddly, I was about to work around the problem by temporarily adding Ejectify to the dock, right-clicking it, going to Options, then checking the option to Open at Login. This, in turn, added Ejectify to the Open at Login section of Login Items in System Settings. I removed Ejectify from the dock and it continues to open on login.

Thank you for your quick response!
